src/NHibernate/Persister/Collection/AbstractCollectionPersister.cs

```diff
@@ -468,7 +471,10 @@ public AbstractCollectionPersister(Mapping.Collection collection, ICacheConcurre
 								   ?? ExecuteUpdateResultCheckStyle.DetermineDefault(collection.CustomSQLUpdate, updateCallable);
 			}
 
+			// 6.0 TODO: call GenerateDeleteRowString(null); instead.
+#pragma warning disable 618
 			sqlDeleteRowString = GenerateDeleteRowString();
```

Comment:
Not replaced yet for avoiding a possible breaking change for those having overridden it.